autoload/lsp/lspserver.vim

```diff
@@ -792,19 +792,37 @@ enddef
 # LSP line and column numbers start from zero, whereas Vim line and column
 # numbers start from one. The LSP column number is the character index in the
 # line and not the byte index in the line.
-def GetLspPosition(): dict<number>
```

Comment:
Can you add some comment here describing the purpose of the "find_ident" argument?

Reply:
take a look and tell me if it makes sense .